certbot-dns-rfc2136/certbot_dns_rfc2136/dns_rfc2136.py

```diff
@@ -206,7 +206,11 @@ def _query_soa(self, domain_name):
         request.flags ^= dns.flags.RD
 
         try:
-            response = dns.query.udp(request, self.server, port=self.port)
+            try:
+                response = dns.query.tcp(request, self.server, port=self.port)
+            except OSError as e:
```

Comment:
Why are we only catching OSErrors here?

From the [source of dns.query.tcp](http://www.dnspython.org/docs/1.16.0/dns.query-pysrc.html#tcp), it looks like other types can be raised.